### PR TITLE
Add attestation events to event stream

### PR DESCRIPTION
**Pull request by cemozerr**
## PR Description
Add attestation events to the event stream. Builds on PR https://github.com/ConsenSys/teku/pull/3477

## Documentation

- [x] I thought about documentation and added the `documentation` label to this PR if updates are required.

## Changelog

- [x] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/attestation/AttestationManager.java [hunk 1]
```diff
@@ -221,6 +236,7 @@ private void sendToSubscribersIfProducedLocally(ValidateableAttestation attestat
     }
 
     notifyAttestationsToSendSubscribers(attestation);
+    notifyAllValidAttestationsSubscribers(attestation);
```
Comment (rolfyone) [inline ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/attestation/AttestationManager.java:239]: one thing we should do is update the open api from the events to remove the statement that we dont support attestation and blocks...

**Review by rolfyone — APPROVED** [review]
LGTM, we just need to cleanup the openapi as said in comments...

File: ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/attestation/AttestationManager.java (hunk 1 above)
Comment (cemozerr) [reply ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/attestation/AttestationManager.java:239]: I was planning to update the changelog and do all the cleaning when I finished the voluntary_exit as well.

Comment (rolfyone) [reply ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/attestation/AttestationManager.java:239]: ok all good.

**Pull request merged**